stackdriver-errors.js

```diff
@@ -13,9 +13,20 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-(function(exports) {
+(function(base, factory) {
```

Comment:
This might be a "common way do things in the UMD JS world", I think it greatly complexifies the code. I am unable to understand what this piece does (or why 10 lines are needed to expose a module). I rather avoid introducing such complexity.